src/wombat.js

```diff
@@ -1655,7 +1655,21 @@ Wombat.prototype.rewriteNodeFuncArgs = function(
   }
   var created = originalFn.call(fnThis, newNode, oldNode);
   if (created && created.tagName === 'IFRAME') {
-    created.allow = 'autoplay \'self\'; fullscreen \'self\'';
+    const basePermissions = {
+      autoplay: '\'self\'',
+      fullscreen: '\'self\'',
+    };
+    const currentPermissions = !created.allow ? {} : Object.fromEntries(
+      created.allow
+        .split(';')
+        .map(value => value.trim().split(' '))
+    );
+
+    created.allow = Object.entries({
```

Comment:
What is the reason for this operation instead of just doing something like:
`created.allow = created.allow ? created.allow + ";" + "") + 'autoplay \'self\'; fullscreen \'self\''`;

Does the above not work?

Reply:
It could be that autoplay and fullscreen are duplicated, but I think that should be ok? Can check if adding them after / before takes precedence.

Reply:
The main reason for this more complex logic is that, just appending `'autoplay \'self\'; fullscreen \'self\''` would overwrite any less restrictive permission that was set for those two directives.

So for example, if it was originally set as `allow="fullscreen *"`, appending that would result in `allow="fullscreen *; autoplay 'self'; fullscreen 'self'"`, where the second `fullscreen` would take precedence over first one, potentially breaking the page in some cases, in a slightly un-intuitive way.

That's why I made the `autoplay` and `fullscreen` directives to be added only if not present already.

That said, it is probably an edge case, and one could even argue that the logic is still not perfect, because if the directive is present but with a more restrictive value, loosening it to `self` might still be required for other wombat features to work.

So I'll leave the final decision to you.

Reply:
Hmm, I think that might be more complex then is needed given the edge case -- could also reverse it so that the wombat overrides come first, eg:
```
created.allow = 'autoplay \'self\'; fullscreen \'self\'' + (created.allow ? created.allow + ";" : "");
```

Maybe can try this and if more complex logic is needed for a particular use case, can add that later?

Reply:
> could also reverse it so that the wombat overrides come first

I think this is a reasonable middle ground which should cover most of the cases. I'll update it.

Reply:
Thanks!